README.md

```diff
@@ -172,7 +172,7 @@ For an overview of the full workflow, see the [workflow diagram](./docs/workflow
       python3 -m venv --upgrade-deps venv
       source venv/bin/activate
       (venv) $ pip cache remove llama_cpp_python
-      (venv) $ pip install instructlab -C cmake.args="-DLLAMA_CUDA=on"
+      (venv) $ pip install instructlab -C cmake.args="-DLLAMA_CUBLAS=on"
```

Comment:
Please undo this change. `LLAMA_CUDA` is the correct option. `LLAMA_CUBLAS` is deprecated.
```suggestion
      (venv) $ pip install instructlab -C cmake.args="-DLLAMA_CUDA=on"
```